frontend/src/shared/getParams.js

```diff
@@ -0,0 +1,10 @@
+import { matchPath } from "react-router-dom";
+
+const getParams = (pathname, path) => {
+  const matchProfile = matchPath(pathname, {
+    path,// specify original path - /stores/:id/orders/:orderId
+  });
+  return (matchProfile && matchProfile.params) || {};
+};
+
+export default getParams;
```

Comment:
I dont understand why we need this instead of using just matchPath or useParams https://reacttraining.com/react-router/web/api/Hooks/useparams

Reply:
I think you misunderstood, or I sent the wrong link, see https://reacttraining.com/react-router/web/api/matchPath and the link above

Reply:
Okay, i thought as much